code/script.js

```diff
@@ -42,46 +41,47 @@ const sunsetIcon = createWeatherImg("./assets/sunset-icon.png", "sunset");
 
 
 function createFiveDayForecast(filteredForecast) {
+    weatherContainer.innerHTML = ""
     for (let i = 0; i < 5; i++) {
 
-        chooseWeatherIcon(filteredForecast, i);
+        const daysWeatherIcon = chooseWeatherIcon(filteredForecast[i]);
         const days = new Date(filteredForecast[i].dt_txt).toLocaleDateString(
             "en-US",
             { weekday: "long" }
         );
         const forecastTemp = Math.round(filteredForecast[i].main.temp * 10) / 10;
         const forecastWeather = filteredForecast[i].weather.description;
-        createFiveDaysInnerHTML(days, forecastTemp);
-    }
+        weatherContainer.innerHTML += createFiveDaysInnerHTML(days, forecastTemp, daysWeatherIcon);
+    };
 };
 
 //Creates a weather icon based on the forecasted weather
-function chooseWeatherIcon(filteredForecast, i) {
-    let main = filteredForecast[i].weather[0].main;
+function chooseWeatherIcon(forecast) {
+    let main = forecast.weather[0].main;
     if (main === "Clouds") {
-        weatherImg = cloudyIcon;
+        return cloudyIcon;
```

Comment:
the function now returns an icon, which we assign to a variable up in the place we called it. (instead of making the assignment inside the function)